modules/vpc/variables.tf

```diff
@@ -0,0 +1,98 @@
+variable "tags" {
+  description = "A mapping of tags to assign to the resources"
+  type        = map(string)
+  default = {
+    terraform = true
+  }
+}
+
+variable "name" {
+  description = "The application name"
+  type        = string
+}
+
+variable "cidr" {
+  description = "The CIDR block for the vpc"
+  type = string
+}
+
+variable "public_subnets" {
+  description = "A list of public subnets inside the VPC"
+  type        = list(string)
+}
+
+variable "private_subnets" {
+  description = "A list of private subnets inside the VPC"
+  type        = list(string)
+}
+
+variable "database_subnets" {
+  description = "A list of database subnets"
+  type        = list(string)
+}
+
+variable "azs" {
+  description = "A list of availability zones names or ids in the region"
+  type = list(string)
+}
+
+variable "create_database_subnet_group" {
+  type    = bool
+  default = true
+}
+
+variable "enable_nat_gateway" {
+  type    = bool
+  default = true
+}
+
+variable "single_nat_gateway" {
+  type    = bool
+  default = false
+}
+
+variable "enable_vpn_gateway" {
+  type    = bool
+  default = false
+}
+
+variable "enable_s3_endpoint" {
+  type    = bool
+  default = false
+}
+
+variable "enable_dhcp_options" {
+  type    = bool
+  default = false
+}
+
+variable "enable_dns_hostnames" {
+  type    = bool
+  default = false
```

Comment:
same question here

Reply:
This one as well. Just set it default to what was default from terraform-aws-modules/vpc/aws module.